packages/alfa-rules/src/sia-r74/rule.ts

```diff
@@ -26,19 +26,14 @@ export default Rule.Atomic.of<Page, Element>({
         return document.elementDescendants(Node.fullTree).filter(
           and(
             hasRole(device, "paragraph"),
-            (element) =>
-              Style.from(element, device)
-                .cascaded("font-size")
-                .some(({ value: fontSize }) => {
-                  switch (fontSize.type) {
-                    case "length":
-                    case "percentage":
-                      return fontSize.value > 0;
-
-                    default:
-                      return true;
-                  }
-                }),
+            hasCascadedStyle(
+              "font-size",
+              (fontSize) =>
+                (fontSize.type === "length" ||
+                  fontSize.type === "percentage") &&
+                fontSize.value > 0,
+              device
+            ),
```

Comment:
This used to return `true` if it was neither length nor percentage, bit now returns `false` 🤔 
(and some test might be missing if we're not catching this 😖 )

Reply:
Right, good catch, what it wanted was probably something like `foo ? fontSize.value > 0 : true`. I'll try and add a failing test and fix it 😄